.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "dpdk"]
 	path = dpdk
-	url = ../dpdk
+	url = https://github.com/ceph/dpdk
```

Comment:
Why is this part necessary?

Reply:
this was needed for my local version since I didn't want to fork that submodule, too, just for the sake of the relative URL to work.
However, this PR itself doesn't require the change, so I can revert it

Reply:
it's a separate commit, actually, so apparently I just pushed too much for the PR. I'll exclude that commit simply